src/CLUBB_core/parameters_tunable.F90

```diff
@@ -518,13 +523,69 @@ subroutine setup_parameters( &
     !lmin = lmin_coef * deltaz  ! Old
     lmin = lmin_coef * lmin_deltaz ! New fixed value
 
-    ! ### Adjust Constant Diffusivity Coefficients Based On Grid Spacing ###
-    call adj_low_res_nu( &
-             nzmax, ngrdcol, grid_type, deltaz,  & ! Intent(in)
-             momentum_heights, thermodynamic_heights, & ! Intent(in)
-             l_prescribed_avg_deltaz, mult_coef, &  ! Intent(in)
-             nu1, nu2, nu6, nu8, nu9, nu10, nu_hm, &  ! Intent(in)
-             nu_vert_res_dep )  ! Intent(out)
+    if ( l_linear_diffusion ) then 
```

Comment:
Does the `l_linear_diffusion` option mean merely that the `c_Kx` parameters are set to zero and the `nu` parameters are set to constants?  If so, does any source code need to be changed at all?

Reply:
Do you want to talk?  I'm on Zoom.  Check your email.